build_tools/android/post_benchmarks_as_pr_comment.py

```diff
@@ -95,24 +135,69 @@ def aggregate_all_benchmarks(
   return sorted([(k,) + v for k, v in aggregate_results.items()])
 
 
-def get_benchmark_result_markdown(benchmark_files: Sequence[str]) -> str:
+def query_base_benchmark_results(commit,
+                                 verbose: bool = False) -> Dict[str, int]:
+  """Queries the benchmark results for the given commit."""
+  build_id = get_git_total_commit_count(commit, verbose)
+
+  url = get_required_env_var('IREE_DASHBOARD_URL')
+  payload = {'projectId': IREE_PROJECT_ID, 'buildId': build_id}
+  return get_from_dashboard(f'{url}/apis/getBuild', payload, verbose=verbose)
+
+
+def get_comparsion_against_base(pr_means: Sequence[int],
+                                base_means: Sequence[int]) -> Sequence[str]:
+  """Returns a tuple of strings comparsing mean latency numbers."""
+  comparisions = []
+
+  for pr, base in zip(pr_means, base_means):
+    if base is None:
+      comparisions.append(str(pr))
+      continue
+
+    if pr > base:
+      percent = "{:.2%}".format((pr - base) / base)
+      direction = "↑"
+    elif pr == base:
+      percent = "{:.2%}".format((pr - base) / base)
+      direction = ""
+    else:
+      percent = "{:.2%}".format((base - pr) / base)
+      direction = "↓"
+    comparisions.append(f"{pr} (vs. {base}, {percent}{direction})")
```

Comment:
Could we emphasize the `↑` / `↓` to make these easier to scan?

---

Current appearance:

| Benchmark Name                                                                                                        | Average Latency (ms)  | Median Latency (ms) | Latency Standard Deviation (ms) |
| --------------------------------------------------------------------------------------------------------------------- | --------------------- | ------------------- | ------------------------------- |
| MobileBertSquad (TensorFlow) fp32-full-inference-1-thread with IREE-Dylib-Sync @ SM-G980F (CPU-ARMv8.2-A)             | 778 (vs. 777, 0.13%↑) | 776                 | 9                               |

---

Maybe use emojis or images?

Time increased: ⬆️🔺🙁
Time decreased: ⬇️🔻🙂

Or colors? https://stackoverflow.com/questions/11509830/how-to-add-color-to-githubs-readme-md-file (multiple options)

Reply:
Excellent points!

Using colors would be the best; it's disappointing that GitHub still does not support it.

I'm using 🚩 and 🎉 for now if the percentage change is larger than a threshold. :)